sdk/eventhub/event-hubs/src/managementClient.ts

```diff
@@ -158,48 +163,52 @@ export class ManagementClient extends LinkEntity {
    * @param connection - The established amqp connection
```

Comment:
Nit: This TSDoc comment is out of date.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/9520/commits/d8620e01c04132efe5462d71eccee607f0e856cd